challenge/src/App.jsx

```diff
@@ -7,7 +7,7 @@ class App extends Component {
     state = {}
     constructor(props) {
         super(props);
-        console.log("Hello from the constructor!");
+        console.log();
```

Comment:
this almost works, i think this still prints a newline.

Reply:
yup